ios/FluentUI/Extensions/UIViewController+Extensions.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+struct ViewControllerHolder {
+    weak var value: UIViewController?
+}
+
+struct ViewControllerKey: EnvironmentKey {
+    static var defaultValue: ViewControllerHolder {
+        return ViewControllerHolder(value: UIApplication.shared.windows.first?.rootViewController)
+    }
+}
+
+public extension EnvironmentValues {
```

Comment:
why is the file name uiviewcontroller+extension when the actual externsion is evnironment values?